src/main/java/org/owasp/wrongsecrets/challenges/docker/Challenge4.java

```diff
@@ -31,7 +31,8 @@ public Spoiler spoiler() {
 
     @Override
     public boolean answerCorrect(String answer) {
-        return argBasedPassword.equals(answer);
+        return argBasedPassword.equals(answer) 
+        || argBasedPassword.equals("'" + answer + "'");
```

Comment:
looks like this is an older MR coming back. This one is already fixed. Can you remove the Challenge4 related changes? as these are already merged to master